source/features/discussion-filters.tsx

```diff
@@ -0,0 +1,47 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import SearchQuery from '../github-helpers/search-query';
+import {getUsername} from '../github-helpers';
+
+function init(): void {
+	// Use an existing dropdown item to preserve its DOM structure (supports old GHE versions)
+	const sourceItem = select<HTMLAnchorElement>([
+		'#filters-select-menu a:nth-last-child(2)', // GHE
+		'.subnav-search-context li:nth-last-child(2)'
+	])!;
+
+	// Add "Everything commented by you" filter
+	const commentsMenuItem = sourceItem.cloneNode(true);
+	const commentsLink = select('a', commentsMenuItem) ?? commentsMenuItem;
+	commentsLink.textContent = 'Everything commented by you';
+	commentsLink.removeAttribute('target');
+	new SearchQuery(commentsLink).set(`is:open commenter:${getUsername()}`);
+
+	sourceItem.after(commentsMenuItem);
+
+	// Add "Everything you subscribed to" link
+	const subscriptionsMenuItem = commentsMenuItem.cloneNode(true);
+	const subscriptionsLink = select('a', subscriptionsMenuItem) ?? subscriptionsMenuItem;
+	subscriptionsLink.textContent = 'Everything you subscribed to';
+	subscriptionsLink.removeAttribute('target');
```

Comment:
I'm a bit unsure if the new "Everything you subscribed to" link should open in a new tab (like the "View advanced search syntax" below it) because there is no easy way back from that page to the repo issue/PR list (apart from the browser's back button of course). If it does, it should look also have that icon and be bold, though, and that looks bad:

![Screenshot_2020-06-09_16-00-01](https://user-images.githubusercontent.com/202916/84156850-4ccd4e00-aa6a-11ea-8732-66da50283f96.png)